### PR TITLE
release: 2.63.1

### PR DESCRIPTION
**Pull request by zyga**
Generated changelogs with:
`DEBEMAIL="Ernest Lotter <ernest.lotter@canonical.com>" release-tools/changelog.py 2.63.1 2061179 NEWS.md`

Primary content:
- [many: update apparmor to 4.0.1]( https://github.com/canonical/snapd/pull/14150) (only support for AppArmor feature min version)
- [interfaces,sandbox: transition steam to permissive apparmor profile](https://github.com/canonical/snapd/pull/14229) 
- [interfaces: switch steam to unrestricted seccomp profile](https://github.com/canonical/snapd/pull/14230)
- [cmd/snap-confine: improve compatibility with nvidia drivers](https://github.com/canonical/snapd/pull/14287)
- [cmd/snap-confine: add libnvidia-vksc-core.so support](https://github.com/canonical/snapd/pull/14306)
- [tests: add integration test for nvidia libs](https://github.com/canonical/snapd/pull/14353)

**Requires rebase-merge.**

### CODE REVIEW

**Review by ernestl — COMMENTED** [review]
Looks good, waiting for latest test results

File: tests/main/snapd-snap/task.yaml [hunk 1]
```diff
@@ -252,8 +252,16 @@ execute: |
     done
 
     echo "Ensure we support posix mqueue and userns in the internal apparmor_parser"
-    snap debug sandbox-features --required apparmor:parser:mqueue
-    snap debug sandbox-features --required apparmor:parser:userns
+    log_bookmark=$(journalctl -u snapd -n 0 --show-cursor | tail -n 1 | awk '{print $3}')
+    # restart to capture AppArmor probe debug information
+    systemctl restart snapd
+    if ! snap debug sandbox-features --required apparmor:parser:mqueue; then
```
Comment (ernestl) [inline tests/main/snapd-snap/task.yaml:258]: This caused failure on: google:ubuntu-20.04-64:tests/main/snapd-snap:lxd

File: tests/main/snapd-homedirs-vendored/task.yaml [hunk 2]
```diff
@@ -18,11 +18,11 @@ prepare: |
     useradd -b /remote/users -m -U "$USERNAME"
 
     # Download current snapd edge
-    snap download --edge snapd --basename=snapd_edge
+    snap download --stable snapd --basename=snapd_stable
```
Comment (ernestl) [inline tests/main/snapd-homedirs-vendored/task.yaml:21]: Intended to fix 
image

File: image/preseed/preseed_linux.go [hunk 3]
```diff
@@ -242,7 +242,19 @@ func prepareCore20Mountpoints(opts *preseedCoreOptions) (cleanupMounts func(), e
 		}
 	}
 
+	underWritable := func(path string) string {
+		return filepath.Join(opts.WritableDir, path)
+	}
+
+	currentLink := underWritable("system-data/snap/snapd/current")
```
Comment (bboozzoo) [inline image/preseed/preseed_linux.go:249]: I'm not sure we need those changes but they shouldn't break anything AFAIU

**Review by bboozzoo — APPROVED** [review]
LGTM

**Review by pedronis — APPROVED** [review]
the set of touched files outside of spread looks right, also the changelog.  As Maciej I'm not sure whether we need the preseed changes because they relate to the new way to build snapd, which I don't think we are backporting but they shouldn't hurt

**Pull request merged**